### PR TITLE
backstage: bump o-tracking internal version

### PR DESCRIPTION
**Pull request by danail-s**
## Describe your changes
Update the internal version of o-tracking

### CODE REVIEW

**Review by FranciscoRuiz-FT — APPROVED** [review]

File: libraries/o-tracking/src/javascript/tracking.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const initPage = page.init;
  *
  * @type {string}
  */
-const version = '4.5.0';
+const version = '4.5.3';
```
Comment (notlee) [inline libraries/o-tracking/src/javascript/tracking.js:20]: o-tracking is currently 4.5.3 but, when this is released, it'll be 4.5.4 😅 

```suggestion
const version = '4.5.4';
```

Comment (notlee) [reply libraries/o-tracking/src/javascript/tracking.js:20]: Oh, right! That's correct. But if we don't release it won't be published to npm, and won't be available to users.

Comment (notlee) [reply libraries/o-tracking/src/javascript/tracking.js:20]: Oh, right! That's correct. However we'll need to release to get the version updated in production. Looks like it's send with tracking events and I guess that's what you want to update?

Comment (danail-s) [reply libraries/o-tracking/src/javascript/tracking.js:20]: Yes, that's correct. Thanks!

**Review by notlee — APPROVED** [review]

File: libraries/o-tracking/src/javascript/tracking.js (hunk 1 above)
Comment (notlee) [reply libraries/o-tracking/src/javascript/tracking.js]: Approved! 👍 I'll go ahead and release

**Pull request merged**